tools/Deployment/gulpfile.js

```diff
@@ -245,7 +250,8 @@ gulp.task("syncBranchCore", () => {
 });
 gulp.task("test", gulp.series("clean", "build", "e2eTest", "publish:myget-test"));
 gulp.task("dev", gulp.series("clean", "build", "e2eTest"));
-gulp.task("dev:release", gulp.series("clean", "build", "e2eTest", "publish:myget-dev", "publish:azdevops-perf-login", "publish:azdevops-perf", "publish:azdevops-internal-login", "publish:azdevops-internal", "publish:azdevops-ppe-login", "publish:azdevops-ppe"));
+gulp.task("dev:release", gulp.series("clean", "build", "e2eTest"));
```

Comment:
We can remove this task as it is same as the above one. Rename to `dev:build`.

Reply:
They will not be same, I am updating.

Reply:
Then, update below one to `dev:release`. It includes pack and publish.

In the next PR, we can use `master:build`, `master:pack`, `master:release` (only for publish).